server.js

```diff
@@ -2,14 +2,14 @@
 const express = require('express');
 const exphbs = require('express-handlebars');
 const sequelize = require('sequelize');
-const db = require('db');
-const {supervisorRouter, employeeRouter, indexRouter, hrRouter} = require('./routes');
+const {employeeRouter, indexRouter} = require('./routes');
```

Comment:
Deleted the other routers here because they didn't exist for me, and as such the server wouldn't run.  Add them back when you have them